suite-common/wallet-core/src/fiat-rates/fiatRatesReducer.ts

```diff
@@ -71,7 +71,7 @@ export const prepareFiatRatesReducer = createReducerWithExtraDeps(
                         state[rateType][fiatRateKey] = {
                             ...currentRate,
                             isLoading: false,
-                            error: result.reason || `Failed to update ${ticker.symbol} fiat rate.`,
+                            error: `${result.reason || `Failed to update ${ticker.symbol} fiat rate.`}`,
```

Comment:
Nit: no need to wrap the entire thing, when the fallback is already string
```suggestion
                            error: String(result.reason) || `Failed to update ${ticker.symbol} fiat rate.`,
```